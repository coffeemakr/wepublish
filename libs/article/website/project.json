{
  "name": "article-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/article/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/article/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/article/website",
        "tsConfig": "libs/article/website/tsconfig.lib.json",
        "packageJson": "libs/article/website/package.json",
        "main": "libs/article/website/src/index.ts",
        "assets": ["libs/article/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/article/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
