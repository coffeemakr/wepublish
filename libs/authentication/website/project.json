{
  "name": "authentication-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/authentication/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/authentication/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/authentication/website",
        "tsConfig": "libs/authentication/website/tsconfig.lib.json",
        "packageJson": "libs/authentication/website/package.json",
        "main": "libs/authentication/website/src/index.ts",
        "assets": ["libs/authentication/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/authentication/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
