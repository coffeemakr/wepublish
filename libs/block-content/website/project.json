{
  "name": "block-content-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/block-content/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/block-content/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/block-content/website",
        "tsConfig": "libs/block-content/website/tsconfig.lib.json",
        "packageJson": "libs/block-content/website/package.json",
        "main": "libs/block-content/website/src/index.ts",
        "assets": ["libs/block-content/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/block-content/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
