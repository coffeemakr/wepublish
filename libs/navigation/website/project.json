{
  "name": "navigation-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/navigation/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/navigation/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/navigation/website",
        "tsConfig": "libs/navigation/website/tsconfig.lib.json",
        "packageJson": "libs/navigation/website/package.json",
        "main": "libs/navigation/website/src/index.ts",
        "assets": ["libs/navigation/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/navigation/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
