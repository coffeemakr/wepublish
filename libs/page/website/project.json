{
  "name": "page-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/page/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/page/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/page/website",
        "tsConfig": "libs/page/website/tsconfig.lib.json",
        "packageJson": "libs/page/website/package.json",
        "main": "libs/page/website/src/index.ts",
        "assets": ["libs/page/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/page/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
