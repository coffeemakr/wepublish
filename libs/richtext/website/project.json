{
  "name": "richtext-website",
  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "libs/richtext/website/src",
  "projectType": "library",
  "tags": [],
  "targets": {
    "lint": {
      "executor": "@nx/linter:eslint",
      "outputs": ["{options.outputFile}"],
      "options": {
        "lintFilePatterns": ["libs/richtext/website/**/*.{ts,tsx,js,jsx}"]
      }
    },
    "build": {
      "executor": "@nx/js:tsc",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/richtext/website",
        "tsConfig": "libs/richtext/website/tsconfig.lib.json",
        "packageJson": "libs/richtext/website/package.json",
        "main": "libs/richtext/website/src/index.ts",
        "assets": ["libs/richtext/website/*.md"],
        "updateBuildableProjectDepsInPackageJson": true
      }
    },
    "test": {
      "executor": "@nx/jest:jest",
      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
      "options": {
        "jestConfig": "libs/richtext/website/jest.config.ts",
        "passWithNoTests": true
      }
    }
  }
}
