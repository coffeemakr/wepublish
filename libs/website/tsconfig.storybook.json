{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "emitDecoratorMetadata": true,
    "outDir": ""
  },
  "files": [
    "../../node_modules/@nx/react/typings/styled-jsx.d.ts",
    "../../node_modules/@nx/react/typings/cssmodule.d.ts",
    "../../node_modules/@nx/react/typings/image.d.ts"
  ],
  "exclude": ["src/**/*.spec.ts", "src/**/*.spec.js", "src/**/*.spec.tsx", "src/**/*.spec.jsx"],
  "include": [
    "../**/website/src/**/*.stories.ts",
    "../**/website/src/**/*.stories.js",
    "../**/website/src/**/*.stories.jsx",
    "../**/website/src/**/*.stories.tsx",
    "../**/website/src/**/*.stories.mdx",
    ".storybook/*.js",
    "preview.tsx",
    "main.ts"
  ]
}
