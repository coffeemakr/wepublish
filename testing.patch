diff --git a/libs/api/src/lib/db/article.ts b/libs/api/src/lib/db/article.ts
index 7359af6a..ea27be09 100644
--- a/libs/api/src/lib/db/article.ts
+++ b/libs/api/src/lib/db/article.ts
@@ -14,11 +14,11 @@ export interface ArticleData {
   readonly lead?: string | null
   readonly seoTitle?: string | null
   readonly slug?: string | null
-  readonly tags: string[]
+  readonly tags?: string[]
 
   readonly canonicalUrl?: string | null
 
-  readonly properties: MetadataProperty[]
+  readonly properties?: MetadataProperty[]
 
   readonly imageID?: string | null
   readonly authors: ArticleRevisionAuthor[]
@@ -107,7 +107,7 @@ export type ArticleRevisionWithRelations = PrismaArticleRevision & {
 
 export type ArticleWithRevisions = PrismaArticle & {
   draft: ArticleRevisionWithRelations | null
-  pending: ArticleRevisionWithRelations | null
+  pending?: ArticleRevisionWithRelations | null
   published: ArticleRevisionWithRelations | null
 }
 
diff --git a/libs/api/src/lib/db/page.ts b/libs/api/src/lib/db/page.ts
index fb73c289..6f0ed1b5 100644
--- a/libs/api/src/lib/db/page.ts
+++ b/libs/api/src/lib/db/page.ts
@@ -14,9 +14,9 @@ export interface PageData {
 
   readonly title: string
   readonly description?: string | null
-  readonly tags: string[]
+  readonly tags?: string[]
 
-  readonly properties: MetadataProperty[]
+  readonly properties?: MetadataProperty[]
 
   readonly imageID?: string | null
 
@@ -85,7 +85,7 @@ export type PageRevisionWithProperties = PrismaPageRevision & {
 
 export type PageWithRevisions = PrismaPage & {
   draft: PageRevisionWithProperties | null
-  pending: PageRevisionWithProperties | null
+  pending?: PageRevisionWithProperties | null
   published: PageRevisionWithProperties | null
 }
 
diff --git a/libs/api/src/lib/graphql/article.ts b/libs/api/src/lib/graphql/article.ts
index 8d80fb27..77357e01 100644
--- a/libs/api/src/lib/graphql/article.ts
+++ b/libs/api/src/lib/graphql/article.ts
@@ -275,7 +275,7 @@ export const GraphQLPublicArticle: GraphQLObjectType<PublicArticle, Context> =
         type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLMetadataPropertyPublic))),
         resolve: ({properties}) => {
           return properties
-            .filter(property => property.public)
+            ?.filter(property => property.public)
             .map(({key, value}) => ({key, value}))
         }
       },
diff --git a/libs/api/src/lib/graphql/article/article.private-mutation.ts b/libs/api/src/lib/graphql/article/article.private-mutation.ts
index 43a6a3ee..049c7e54 100644
--- a/libs/api/src/lib/graphql/article/article.private-mutation.ts
+++ b/libs/api/src/lib/graphql/article/article.private-mutation.ts
@@ -178,7 +178,8 @@ export const duplicateArticle = async (
       createMany: {
         data: socialMediaAuthors.map(({authorId}) => ({authorId}))
       }
-    }
+    },
+    blocks: []
   }
 
   return articleClient.create({
@@ -303,7 +304,8 @@ export const unpublishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -333,7 +335,8 @@ export const unpublishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           }
         }
       },
@@ -491,7 +494,8 @@ export const publishArticle = async (
                 createMany: {
                   data: socialMediaAuthors.map(({authorId}) => ({authorId}))
                 }
-              }
+              },
+              blocks: []
             },
             update: {
               ...revision,
@@ -521,7 +525,8 @@ export const publishArticle = async (
                 createMany: {
                   data: socialMediaAuthors.map(({authorId}) => ({authorId}))
                 }
-              }
+              },
+              blocks: []
             }
           }
         },
@@ -579,7 +584,8 @@ export const publishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -609,7 +615,8 @@ export const publishArticle = async (
               createMany: {
                 data: socialMediaAuthors.map(({authorId}) => ({authorId}))
               }
-            }
+            },
+            blocks: []
           }
         }
       },
diff --git a/libs/api/src/lib/graphql/page.ts b/libs/api/src/lib/graphql/page.ts
index 56d380f0..6b63518e 100644
--- a/libs/api/src/lib/graphql/page.ts
+++ b/libs/api/src/lib/graphql/page.ts
@@ -210,7 +210,7 @@ export const GraphQLPublicPage = new GraphQLObjectType<PublicPage, Context>({
     properties: {
       type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLMetadataPropertyPublic))),
       resolve: ({properties}) => {
-        return properties.filter(property => property.public).map(({key, value}) => ({key, value}))
+        return properties?.filter(property => property.public).map(({key, value}) => ({key, value}))
       }
     },
 
diff --git a/libs/api/src/lib/graphql/page/page.private-mutation.ts b/libs/api/src/lib/graphql/page/page.private-mutation.ts
index 8d21a995..456df84e 100644
--- a/libs/api/src/lib/graphql/page/page.private-mutation.ts
+++ b/libs/api/src/lib/graphql/page/page.private-mutation.ts
@@ -140,7 +140,8 @@ export const duplicatePage = async (
       createMany: {
         data: duplicatedProperties
       }
-    }
+    },
+    blocks: []
   }
 
   return pageClient.create({
@@ -236,7 +237,8 @@ export const unpublishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -250,7 +252,8 @@ export const unpublishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           }
         }
       },
@@ -387,7 +390,8 @@ export const publishPage = async (
                 createMany: {
                   data: properties
                 }
-              }
+              },
+              blocks: []
             },
             update: {
               ...revision,
@@ -401,7 +405,8 @@ export const publishPage = async (
                 createMany: {
                   data: properties
                 }
-              }
+              },
+              blocks: []
             }
           }
         },
@@ -443,7 +448,8 @@ export const publishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           },
           update: {
             ...revision,
@@ -454,7 +460,8 @@ export const publishPage = async (
               createMany: {
                 data: properties
               }
-            }
+            },
+            blocks: []
           }
         }
       },
