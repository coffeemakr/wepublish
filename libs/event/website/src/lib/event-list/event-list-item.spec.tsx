import {runStorybookTests} from '@wepublish/testing/website'
import * as stories from './event-list-item.stories'

describe('EventListItem', () => {
  runStorybookTests(stories)
})
