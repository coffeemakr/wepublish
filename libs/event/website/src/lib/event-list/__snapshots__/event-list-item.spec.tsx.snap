// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventListItem should render Default 1`] = `
<DocumentFragment>
  .emotion-0 {
  margin: 0;
  font: inherit;
  color: #1976d2;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  text-decoration-color: rgba(25, 118, 210, 0.4);
  width: 100%;
  color: inherit;
  -webkit-text-decoration: inherit;
  text-decoration: inherit;
}

.emotion-0:hover {
  text-decoration-color: inherit;
}

.emotion-1 {
  display: grid;
  gap: 16px;
  grid-template-columns: 1fr;
  -webkit-align-content: start;
  -ms-flex-line-pack: start;
  align-content: start;
}

@media (min-width:900px) {
  .emotion-1 {
    grid-template-columns: minmax(30%, 200px) auto;
  }
}

.emotion-5 {
  max-width: 100%;
  width: 100%;
  height: auto;
  aspect-ratio: auto 0.6666666666666666;
}

.emotion-7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex-flow: column;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  gap: 8px;
}

.emotion-9 {
  display: grid;
  grid-template-columns: auto auto auto;
  gap: 8px;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-11 {
  display: grid;
  grid-template-columns: auto auto;
  gap: 8px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-13 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 2.125rem;
  line-height: 1.235;
  letter-spacing: 0.00735em;
}

.emotion-16 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5;
  letter-spacing: 0.00938em;
  margin-bottom: 0.35em;
}

<a
    class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways emotion-0"
    href="https://example.com"
  >
    <article
      class="emotion-1 emotion-2"
    >
      <div
        class="emotion-3 emotion-4"
      >
        <img
          alt="DSC07717"
          class="emotion-5 emotion-6"
          loading="lazy"
          srcset="https://unsplash.it/800/400 800w,
https://unsplash.it/500/300 500w,
https://unsplash.it/300/200 300w,
https://unsplash.it/200/100 200w"
          title=""
        />
      </div>
      <div
        class="emotion-7 emotion-8"
      >
        <div
          class="emotion-9 emotion-10"
        >
          <div
            class="emotion-11 emotion-12"
          >
            <svg
              fill="currentColor"
              height="1em"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M9 11H7v2h2v-2zm4 0h-2v2h2v-2zm4 0h-2v2h2v-2zm2-7h-1V2h-2v2H8V2H6v2H5c-1.11 0-1.99.9-1.99 2L3 20a2 2 0 002 2h14c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 16H5V9h14v11z"
              />
            </svg>
            <time
              datetime="2023-02-24T09:00:00.000Z"
            >
              24.02.2023 09:00
            </time>
          </div>
          â€“ 
          <time
            datetime="2023-02-25T07:30:00.000Z"
          >
            25.02.2023 07:30
          </time>
        </div>
        <h1
          class="MuiTypography-root MuiTypography-h4 emotion-13"
        >
          Cool concert
        </h1>
        <div
          class="emotion-14 emotion-15"
        >
          <div
            data-gramm="false"
            data-slate-editor="true"
            data-slate-node="value"
            style="outline: none; white-space: pre-wrap; word-wrap: break-word;"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom emotion-16"
              data-slate-node="element"
            >
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Lorem ipsum dolor sit amet, 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <strong
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    consectetur adipiscing elit, 
                  </span>
                </strong>
              </span>
              <span
                data-slate-node="text"
              >
                <em
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
                  </span>
                </em>
              </span>
              <span
                data-slate-node="text"
              >
                <u
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    Ut enim ad minim veniam, 
                  </span>
                </u>
              </span>
              <span
                data-slate-node="text"
              >
                <del
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
                  </span>
                </del>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                </span>
              </span>
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom emotion-16"
              data-slate-node="element"
            >
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Lorem ipsum dolor sit amet, 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <strong
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    consectetur adipiscing elit, 
                  </span>
                </strong>
              </span>
              <span
                data-slate-node="text"
              >
                <em
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
                  </span>
                </em>
              </span>
              <span
                data-slate-node="text"
              >
                <u
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    Ut enim ad minim veniam, 
                  </span>
                </u>
              </span>
              <span
                data-slate-node="text"
              >
                <del
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
                  </span>
                </del>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                </span>
              </span>
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom emotion-16"
              data-slate-node="element"
            >
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Lorem ipsum dolor sit amet, 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <strong
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    consectetur adipiscing elit, 
                  </span>
                </strong>
              </span>
              <span
                data-slate-node="text"
              >
                <em
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
                  </span>
                </em>
              </span>
              <span
                data-slate-node="text"
              >
                <u
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    Ut enim ad minim veniam, 
                  </span>
                </u>
              </span>
              <span
                data-slate-node="text"
              >
                <del
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
                  </span>
                </del>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                </span>
              </span>
            </p>
            <p
              class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom emotion-16"
              data-slate-node="element"
            >
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Lorem ipsum dolor sit amet, 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <strong
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    consectetur adipiscing elit, 
                  </span>
                </strong>
              </span>
              <span
                data-slate-node="text"
              >
                <em
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
                  </span>
                </em>
              </span>
              <span
                data-slate-node="text"
              >
                <u
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    Ut enim ad minim veniam, 
                  </span>
                </u>
              </span>
              <span
                data-slate-node="text"
              >
                <del
                  data-slate-leaf="true"
                >
                  <span
                    data-slate-string="true"
                  >
                    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
                  </span>
                </del>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
                </span>
              </span>
              <span
                data-slate-node="text"
              >
                <span
                  data-slate-string="true"
                >
                  Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
                </span>
              </span>
            </p>
          </div>
        </div>
      </div>
    </article>
  </a>
</DocumentFragment>
`;

exports[`EventListItem should render WithClassName 1`] = `
"Snapshot Diff:
- First value
+ Second value

@@ -2,11 +2,11 @@
    <a
      class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-4wstp6-MuiTypography-root-MuiLink-root-EventListItem"
      href="https://example.com"
    >
      <article
-       class="css-ki8is6-EventListItemWrapper ehub4u74"
+       class="extra-classname css-ki8is6-EventListItemWrapper ehub4u74"
      >
        <div
          class="css-13xyioz-EventListItemImage ehub4u70"
        >
          <img"
`;

exports[`EventListItem should render WithEmotion 1`] = `
"Snapshot Diff:
Compared values have no visual difference."
`;

exports[`EventListItem should render WithoutImage 1`] = `
"Snapshot Diff:
- First value
+ Second value

@@ -6,22 +6,11 @@
      <article
        class="css-ki8is6-EventListItemWrapper ehub4u74"
      >
        <div
          class="css-13xyioz-EventListItemImage ehub4u70"
-       >
-         <img
-           alt="DSC07717"
-           class="css-1sg8o3u-ImageWrapper e1shv2yg0"
-           loading="lazy"
-           srcset="https://unsplash.it/800/400 800w,
- https://unsplash.it/500/300 500w,
- https://unsplash.it/300/200 300w,
- https://unsplash.it/200/100 200w"
-           title=""
        />
-       </div>
        <div
          class="css-13m43v7-EventListItemContent ehub4u73"
        >
          <div
            class="css-3m8h5i-EventListItemMeta ehub4u72""
`;
